test/e2e/conformance_test.go

```diff
@@ -159,31 +163,30 @@ var _ = Describe("Conformance Tests", func() {
 		controlPlaneMachineCount, err := strconv.ParseInt(e2eConfig.GetVariable("CONFORMANCE_CONTROL_PLANE_MACHINE_COUNT"), 10, 64)
 		Expect(err).NotTo(HaveOccurred())
 
-		runtime := b.Time("cluster creation", func() {
-			clusterctl.ApplyClusterTemplateAndWait(ctx, clusterctl.ApplyClusterTemplateAndWaitInput{
-				ClusterProxy: bootstrapClusterProxy,
-				ConfigCluster: clusterctl.ConfigClusterInput{
-					LogFolder:                filepath.Join(artifactFolder, "clusters", bootstrapClusterProxy.GetName()),
-					ClusterctlConfigPath:     clusterctlConfigPath,
-					KubeconfigPath:           bootstrapClusterProxy.GetKubeconfigPath(),
-					InfrastructureProvider:   clusterctl.DefaultInfrastructureProvider,
-					Flavor:                   flavor,
-					Namespace:                namespace.Name,
-					ClusterName:              clusterName,
-					KubernetesVersion:        kubernetesVersion,
-					ControlPlaneMachineCount: pointer.Int64Ptr(controlPlaneMachineCount),
-					WorkerMachineCount:       pointer.Int64Ptr(linuxWorkerMachineCount),
-				},
-				WaitForClusterIntervals:      e2eConfig.GetIntervals(specName, "wait-cluster"),
-				WaitForControlPlaneIntervals: e2eConfig.GetIntervals(specName, "wait-control-plane"),
-				WaitForMachineDeployments:    e2eConfig.GetIntervals(specName, "wait-worker-nodes"),
-				ControlPlaneWaiters: clusterctl.ControlPlaneWaiters{
-					WaitForControlPlaneInitialized: EnsureControlPlaneInitialized,
-				},
-			}, result)
-		})
-
-		b.RecordValue("cluster creation", runtime.Seconds())
+		stopwatch := experiment.NewStopwatch()
+		clusterctl.ApplyClusterTemplateAndWait(ctx, clusterctl.ApplyClusterTemplateAndWaitInput{
+			ClusterProxy: bootstrapClusterProxy,
+			ConfigCluster: clusterctl.ConfigClusterInput{
+				LogFolder:                filepath.Join(artifactFolder, "clusters", bootstrapClusterProxy.GetName()),
+				ClusterctlConfigPath:     clusterctlConfigPath,
+				KubeconfigPath:           bootstrapClusterProxy.GetKubeconfigPath(),
+				InfrastructureProvider:   clusterctl.DefaultInfrastructureProvider,
+				Flavor:                   flavor,
+				Namespace:                namespace.Name,
+				ClusterName:              clusterName,
+				KubernetesVersion:        kubernetesVersion,
+				ControlPlaneMachineCount: pointer.Int64Ptr(controlPlaneMachineCount),
+				WorkerMachineCount:       pointer.Int64Ptr(linuxWorkerMachineCount),
+			},
+			WaitForClusterIntervals:      e2eConfig.GetIntervals(specName, "wait-cluster"),
+			WaitForControlPlaneIntervals: e2eConfig.GetIntervals(specName, "wait-control-plane"),
+			WaitForMachineDeployments:    e2eConfig.GetIntervals(specName, "wait-worker-nodes"),
+			ControlPlaneWaiters: clusterctl.ControlPlaneWaiters{
+				WaitForControlPlaneInitialized: EnsureControlPlaneInitialized,
+			},
+		}, result)
+		stopwatch.Record("cluster creation")
```

Comment:
I don't see anything in output about "cluster creation" or "conformance suite", is that supposed to appear as output somewhere?

https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_cluster-api-provider-azure/2941/pull-cluster-api-provider-azure-conformance/1603444304154464256/build-log.txt

Reply:
Yes, it's *supposed* to by virtue of the `AddReportEntry` function call: https://onsi.github.io/ginkgo/#attaching-data-to-reports

Reply:
I too see neither of those strings in the logs. 😕

Reply:
By searching for `conformance-tests`, I see where `AddReportEntry` was called at around line 500 in the logs, but I don't see the contents.

```
  INFO: Creating namespace capz-conf-w0d1ke
  INFO: Creating event watcher for namespace "capz-conf-w0d1ke"
  conformance-tests - /home/prow/go/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/conformance_test.go:102 @ 12/15/22 20:46:08.684
    conformance-tests
    Name | N | Min | Median | Mean | StdDev | Max
  INFO: Creating the workload cluster with name "capz-conf-w0d1ke" using the "(default)" template (Kubernetes v1.24.9, 1 control-plane machines, 2 worker machines)
```

 Maybe an `experiment` doesn't [implement `fmt.Stringer`](https://onsi.github.io/ginkgo/#controlling-output) (which would be rude since it's recommended in code examples) or maybe we need to log the report entries manually, in `AfterEach`? The tests haven't run yet, so there's nothing really to log, but this is the recommended pattern... I'll dig around more.

Reply:
Since this is fixing the original issue (the tests don't run at all anymore) which is a lot higher priority to fix than logging the duration in the output (IMO), how about we merge this as-is and open an issue to investigate the logging?

Reply:
I think that's reasonable. Let me pare this back to the original commit and open an issue.

Reply:
+1

Reply:
See #2951